packages/react/src/Link/Link.tsx

```diff
@@ -6,7 +6,14 @@ import {Text} from '../Text'
 import styles from './Link.module.css'
 
 export type LinkProps = {
+  /**
+   * The size variations available in Link
+   */
   size?: 'medium' | 'large'
+  /**
+   * Displays the arrow icon in the opposite direction to the default
+   */
+  reverseArrow?: boolean
```

Comment:
How do you feel about calling this `arrowDirection` and the value could be either be `left` or `right`?

Reply:
I like that idea! It's much better for internationalisation too, as we're not implying one is more correct than the other, particularly in languages that read RTL.

I've changed to arrowDirection 👍